docs/docs/end-users/Configuration.fsx

```diff
@@ -0,0 +1,166 @@
+(**
+---
+category: End-users
+categoryindex: 1
+index: 3
+---
+*)
+#r "nuget: Fantomas.Core, 5.0.0-alpha-*"
+
+open Fantomas.Core.FormatConfig
+open Fantomas.Core
+
+(**
+# Configuration
+Fantomas ships with a series of format options.
+These can be stored in an [.editorconfig](https://editorconfig.org/) file and will be picked up automatically by the 
+commandline. <br>
+Your IDE should respect your settings, but the implementation of that is editor specific. Setting the configuration via 
+UI might be available depending on the IDE but do note that these features might not be up to 
+date nor work properly, therefore it's support and use is out of scope for this project. 
+*)
+
+(**
+## Usage
+Inside .editorconfig you can specify the file extension and code location to be use per config:
+```
+[*.fs]
+fsharp_space_before_uppercase_invocation = true
+
+# Write a comment by starting the line with a '#'
+[*.{fs,fsx,fsi}]
+fsharp_bar_before_discriminated_union_declaration = true
+
+# Apply specific settings for a targeted subfolder
+[src/Elmish/View.fs]
+fsharp_multiline_block_brackets_on_same_column = true
+fsharp_experimental_stroustrup_style = true
+```
+*)
+
+(**
+## Trying your settings via the online tool
+You can quickly try your settings via the <a href="https://fsprojects.github.io/fantomas-tools/#/fantomas/preview" target="_blank">online tool</a>. <br>
+<img src="{{root}}/online_tool_usage.gif" alt="drawing" width="100%"/>
+*)
+
+(**
+## Settings recommendations
+Fantomas ships with a series of settings that you can use freely depending  on your case. However there are settings that we do not recommend and generally should not be used. <br>
```

Comment:
`However,`

The `<br>` can typically be avoided in markdown by adding two trailing spaces at the end of the sentence.